core/geometry/src/polyface/PolyfaceBuilder.ts

```diff
@@ -112,8 +112,8 @@ class FacetSector {
    */
   public loadIndexedPointAndDerivativeCoordinatesFromPackedArrays(i: number, packedXYZ: GrowableXYZArray, packedDerivatives?: GrowableXYZArray, fractions?: GrowableFloat64Array, v?: number) {
     packedXYZ.getPoint3dAtCheckedPointIndex(i, this.xyz);
-    if (fractions && v !== undefined)
-      this.uv = Point2d.create(fractions.atUncheckedIndex(i), v);
+    if (fractions && v !== undefined) { }
+    // this.uv = Point2d.create(fractions.atUncheckedIndex(i), v);
```

Comment:
Dave please take a look at this.

Note that commenting out this line fixes the test. However, we still hit
```      
    if (messages.length > 0) {
        this.data.trimAllIndexArrays(lengthA);
        return messages;
      }
```
in `terminateFacet` with messages such as "Less than 3 indices in the last facet" and  "invalid point indices in the last facet".

Reply:
Addressed!